.travis.yml

```diff
@@ -2,11 +2,19 @@ language: go
 go:
   - 1.8.3
   - 1.9.2
+branches:
```

Comment:
could you drop 1.8 and add "1.10.x" (the quotes matter)

Reply:
Sure thing.